submissions/vovan-zt/Memory-Pair-Game/script.js

```diff
@@ -0,0 +1,131 @@
+window.addEventListener('DOMContentLoaded', () => {
+
+    const cards = [
+        {
+            name: 'card1',
+            image: 'img/1.png'
+        },
+        {
+            name: 'card1',
+            image: 'img/1.png'
+        },
+        {
+            name: 'card2',
+            image: 'img/2.png'
+        },
+        {
+            name: 'card2',
+            image: 'img/2.png'
+        },
+        {
+            name: 'card3',
+            image: 'img/3.png'
+        },
+        {
+            name: 'card3',
+            image: 'img/3.png'
+        },
+        {
+            name: 'card4',
+            image: 'img/4.png'
+        },
+        {
+            name: 'card4',
+            image: 'img/4.png'
+        },
+        {
+            name: 'card5',
+            image: 'img/5.png'
+        },
+        {
+            name: 'card5',
+            image: 'img/5.png'
+        },
+        {
+            name: 'card6',
+            image: 'img/6.png'
+        },
+        {
+            name: 'card6',
+            image: 'img/6.png'
+        },
+        
+    ];
+
+    const wrapper = document.querySelector('.wrapper');
+
+    cards.sort(function() { return 0.5 - Math.random() });
+    
+  
+    cards.forEach(item => {
+        const element = document.createElement('div');
+        element.classList.add('col'); 
+        element.innerHTML = `
+                <div class="c1" data-card='${item.name}'></div>
+                <img class="c2" src="${item.image}" alt="">
+
+        `
+        wrapper.append(element);
+    })                   
+
+    const flip = document.querySelectorAll('.col');
+    let cardFirst = [];
+
+    function cancell() {
+        flip.forEach(item => { 
+            item.classList.remove('hover');
+        })
+    }
+
+    console.log(flip);
+
+    function visibility() {
+        flip.forEach(item => { 
+            item.classList.remove('visibility');
+        })
+    }
+
+    function reset () {
+        let count = 0;
+        flip.forEach (item => {
+            if (item.classList.contains('visibility')) {
```

Comment:
1. `querySelectorAll('visibility').length`
2. But better will be if you will keep this state in JS code somewhere

Reply:
remade